content/projects/gsoc.adoc

```diff
@@ -13,12 +13,14 @@ This page is about Google Summer of Code (GSoC) 2017.
 Information about the previous year is archived and referenced below.
 
 Preparation is still underway for GSoC 2017; the project has not been accepted yet.
-The list of accepted organizations will be published on **Feb 27**.
+The list of accepted organizations will be published on **Feb 28**.
 All students are welcome to start project discussions before this date.
 ====
 
 == About GSoC
 
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
```

Comment:
Maybe add to https://wiki.jenkins-ci.org/display/JENKINS/Logo for now? No strong preference though.